content/docs/authorization/cli.md

```diff
@@ -615,7 +617,9 @@ karavictl storage [flags]
 ##### Options
 
 ```
-  -h, --help   help for storage
+  --addr string   address of the csm-authorization storage service
+  -h, --help      help for storage
+  --insecure      insecure skip verify
```

Comment:
If this insecure flag serves the same purpose as the insecure flag for storage, lets keep the description consistent -  `Skip certificate validation`